src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -41,6 +41,7 @@ public static function getProvidedTypes()
             'float' => 'float',
             'int' => 'int',
             'json' => 'array',
+            'key' => 'string',
```

Comment:
It could be anything, like const

Reply:
const is not allowed in the returned types (or at least, that is what the tests tell me when I add it). A const in only a const when the processor has interpreted a string as such right?

Would updated this to `'key' => 'bool|int|float|string|array',` make sense?

Reply:
it would to me yes

Reply:
That's a verb, all the other are nouns...

Reply:
`resolve` :)

Reply:
`resolve` is already used to resolve kernel parameters in the env var

Reply:
Sorry, i meant "resolve" is also a verb.. legitimating to use "fetch" here.

Reply:
Right, of course... Should have seen that :) I think key is better as you are actually specifying the key you want to select from an array, while fetch can actually be to fetch right about anything.

Reply:
I prefer "key" as well